### PR TITLE
feat: MapLibre Android SDK 11.5.0

### PR DESCRIPTION
**Pull request by tyrauber**
## Description

Update MapLibre Android SDK to 11.5.0

## Checklist

- [x] I have tested this on a device/simulator for each compatible OS
- [x] I formatted JS and TS files with running `yarn lint:fix` in the root folder
- [x] I have run tests via `yarn test` in the root folder
- [x] I updated the documentation with running `yarn generate` in the root folder
- [x] I mentioned this change in `CHANGELOG.md`
- [ ] I updated the typings files (`index.d.ts`)
- [ ] I added/updated a sample (`/example`)

### CODE REVIEW

**Comment by caspg** [comment]
Changes are looking good. We should test it though, on simulator or device.

**Comment by tyrauber** [comment]
@caspg, thanks for the review! Builds for both react-native-app and expo-app for ios and android.  Cutting as alpha.19.

File: javascript/utils/MaplibreStyles.d.ts [hunk 1]
```diff
@@ -556,14 +553,11 @@ export interface LineLayerStyleProps {
...
-  lineTranslateAnchor?: Value<
-    Enum<LineTranslateAnchorEnum, LineTranslateAnchorEnumValues>,
-    ["zoom"]
-  >;
+  lineTranslateAnchor?: "map" | "viewport";
   /**
    * Stroke thickness.
    */
-  lineWidth?: Value<number, ["zoom", "feature", "feature-state"]>;
+  lineWidth?: number;
```
Comment (KiwiKilian) [inline javascript/utils/MaplibreStyles.d.ts:560]: Did you notice this breaks a lot of expression styles? @tyrauber @caspg

Comment (tyrauber) [reply javascript/utils/MaplibreStyles.d.ts:560]: I did not.  This file is programmatically generated from the published style-spec. So these changes were made upstream.  I was able to build for both iOS and Android, and our test suite failed to detect any issues.  Can you give me specifics.  I can revert this change, but the Android SDK requires an up-to-date style-spec.

**Pull request merged**